shared/CloudBackupAndRestoreUtils.ts

```diff
@@ -431,6 +498,7 @@ export type isSignedInResult = {
   isSignedIn: boolean;
   error?: string | null;
   profileInfo?: ProfileInfo;
+  isAuthorisedd?: boolean;
```

Comment:
Can we rename this to isAuthorised, to avoid spelling issues?

Reply:
renamed